src/Components/banner.js

```diff
@@ -0,0 +1,14 @@
+import React from "react";
+
+const Banner = (props) => {
```

Comment:
Just a note that we should rename this file to capitalized for file naming consistency with its contents... Banner.js instead of banner.js... note that the Mac filesystem is actually NOT case sensitive, which causes weird issues in renaming- you can't just capitalize the B, you'll need to rename the file to something completely different, then back to Banner.js